driver/normalizer/annotation.go

```diff
@@ -87,5 +131,47 @@ var AnnotationRules = On(Any).Self(
 			On(HasProperty("Tok", "goto")).Roles(uast.Goto),
 			// TODO: fallthrough
 		),
+
+		// Switch statement
+		On(goast.SwitchStmt).Roles(uast.Switch),
+		On(goast.CaseClause).Roles(uast.Case),
+		// TODO: default (length of List is zero)
+
+		// Declarations
+		On(goast.GenDecl).Roles(uast.Declaration).Self(
+			On(HasProperty("Tok", "var")).Roles(uast.Variable).Children(
```

Comment:
Mmmm, three anidated nodes with the same role usually mean that at least some of them have Incomplete (uast.Incomplete if no other roles can apply) information.

Reply:
Removed central annotation. `GenDecl` will be `Variable,Declaration,Statement` and `Name` will be `Variable,Identifier,Expression`.